src/programs/candyMachine/accounts/CandyMachineAccount.ts

```diff
@@ -1,62 +1,6 @@
-import { CandyMachine, Creator } from '@metaplex-foundation/mpl-candy-machine';
-import { BaseAccount, UnparsedAccount, UnparsedMaybeAccount } from '@/types';
-import {
-  assertCreators,
-  assertName,
-  assertSymbol,
-  assertUri,
-  getConfigLines,
-  getConfigLinesCount,
-  getSpaceForCandy,
-} from './candyMachineInternals';
+import { CandyMachine } from '@metaplex-foundation/mpl-candy-machine';
+import { Account, getAccountParsingFunction } from '@/types';
 
-export class CandyMachineAccount extends BaseAccount<CandyMachine> {
-  get size() {
-    return getSpaceForCandy(this.data.data);
-  }
+export type CandyMachineAccount = Account<CandyMachine>;
 
-  static getConfigLinesCount(rawData: Buffer) {
```

Comment:
I had those as static methods to ease discoverability ... not sure how easy it is now to find those methods given that this package exports tons of things already.
Therefore grouping via class might not be the worst.

Could we find a way to make this easier to discover by grouping exports maybe?